doc/DeveloperGuide.md

```diff
@@ -55,10 +55,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
+`* * *` | new user | import my address book | continue using it
```

Comment:
Would a `new user` have existing address book?
Perhaps, import contacts from other platform would be more appropriate.